server/README.md

```diff
@@ -0,0 +1,22 @@
+# TACC Frontera Web Portal - Server
+
+## Directories
+
+- _`cms`: Isolate assets for the CMS website._
+- `conf`: Configure services (like CMS, Nginx, and Docker that runs them).
+- _`docs`: Isolate assets for the Tech Docs / User Guide._
+- _`media`: Dumped assets for the temporary CMS in which the Portal runs._
+- `portal`: Isolate code for the Portal.
```

Comment:
```suggestion
- `portal`: Isolate code for the Portal's Django backend.
```

Reply:
I had to think a bit on this:
- __Yes, "Portal" is not accurate enough.__ Portal can means "not the CMS" to CMS developers; Portal can mean "the CMS and the webapp and the Django backend" to management and consumers.
- __No, "backend" is unnecessary.__ The user is reading a __`/server`__`/README.md`, so they must be aware that they are in backend code.
- __No, "Django" is unnecessary.__ Keep it high-level. As the user does not need to know that the CMS is Django, so does the user not need to know that the backend is Django. There can be a `/server/portal/README.md` to explain that.

I welcome an alternate proposal. I am not keen enough to think of the words, but—as I currently evaluate—this suggestion offers two negatives for one positive.

Reply:
The use of `portal` here: is that A) isolate code specific to portal projects (such as custom branding, styles, templates, etc) or B) isolate code specific to the portal (as in the web app + django) in a general sense, but used by all portals?

If it is A, then something like `project` may be more appropriate. If it is B, `portal` seems perfectly descriptive. 

If it is neither, please elaborate further... =)

Reply:
Ok, then how about `Portal's backend`. I think the distinction is important, and is more informative for readers who may not immediately equate `server` to `backend`.

Reply:
And @taoteg it is C) isolate code for the portal backend specifically, as the frontend webapp portal code lives in `client`

Reply:
Renamed from `Portal` to `Portal's backend`. https://github.com/TACC/Frontera-Portal/pull/60/commits/95510376dff6aeb6b82cc7210b0a9762ba8547e7